image_load.py

```diff
@@ -0,0 +1,59 @@
+from pexelsapi.pexels import Pexels
+import requests
+import logging
+import random
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("Image_Load_Logger")
+
+
+class ImageException(Exception):
+    pass
+
+
+class APINotAvailable(ImageException):
+    pass
+
+
+class PexelAPIKeyError(ImageException):
+    pass
+
+
+def get_image_url(pexel_api_key):
+    try:
+        logger.info('Connecting to Pexels API...')
+        pexel = Pexels(pexel_api_key)
+        search_photos = pexel.curated_photos(page=1, per_page=80)
+        logger.info('Getting image url...')
+        image_url = search_photos['photos'][random.randint(0, 79)]['src']['tiny']
+        logger.info('Image url received')
+
+        return image_url
+
+    except requests.exceptions.JSONDecodeError as e:

```

Comment:
I assume you used `pexels-api-py` for getting an image, and it is bad implemented IMO, so it leads to very confusing exception handling here, it is very strange to see `apinotavailable` on `jsondecodeerror` (because well, `jsondecodeerror` corresponds to errors appeared while decoding some json, so I would assume, some answer was received from api, but looks like not in this case). I would suggest to implement your own method to receive image from api of this resource.

Reply:
Yes, it's just working in that way after testing, but it's really looking not clear, so at least i can just add response check first, and leave JSONDecodeError only for cases of problems with decoding received json